package.json

```diff
@@ -78,6 +78,7 @@
     "fast-safe-stringify": "^2.0.6",
     "http-headers": "^3.0.2",
     "is-native": "^1.0.1",
+    "opentracing": "^0.14.3",
```

Comment:
@Qard I'm not a big fan of having a dependency to opentracing here as most people probably don't need it. Alternatively the entire opentracing logic could be moved to a separate module - or something else. What do you think?

Reply:
Depends how much opentracing adds to the tree. If it's a minimal delta, it's probably fine to just have it as a dependency. If you feel we'd be better off maintaining it as a separate module, I'd be fine with that too though.

Reply:
The `opentracing` package doesn't it self have any dependencies. Here's [its stats](https://bundlephobia.com/result?p=opentracing@0.14.3):

![image](https://user-images.githubusercontent.com/10602/49374675-5c940580-f702-11e8-9e74-6e3a10fd4da0.png)

So I guess it's ok for us to add it